library/std/src/keyword_docs.rs

```diff
@@ -1522,7 +1516,7 @@ mod static_keyword {}
 /// For more information on structs, take a look at the [Rust Book][book] or the
 /// [Reference][reference].
 ///
-/// [`PhantomData`]: marker/struct.PhantomData.html
+/// [`PhantomData`]: marker::PhantomData
```

Comment:
`marker` is not in scope.
```suggestion
/// [`PhantomData`]: crate::marker::PhantomData
```